app/models/post.rb

```diff
@@ -1,12 +1,12 @@
 class Post < ApplicationRecord
   belongs_to :user
   belongs_to :book
-  has_many :comments
-  has_many :pictures
+  has_many :comments, dependent: :destroy
+  has_many :pictures, dependent: :destroy
 
   delegate :name, to: :user, prefix: :user
 
   scope :order_desc, ->{order created_at: :desc}
 
-  enum status: [:locked, :opened]
+  enum status: {locked: 0, opened: 1}
```

Comment:
Tại sao em viết như thế này?

Reply:
rubocop nhắc sửa thế ạ

Reply:
Em tìm hiểu xem tại sao rubocop bảo sửa thế này

Reply:
nếu mà để như trước ý ạ, thì mình sẽ lấy giá trị của enum theo thứ tự của nó trong mảng, thì ví dụ nếu mình muốn chèn 1 trạng thái đằng trước locked thì sẽ ảnh hưởng đến toàn bộ bọn đằng sau ạ... còn nếu để dạng hash này thì em đang gán cho nó bằng 0 và 1 nên thêm bớt các trạng thái sẽ không bị ảnh hưởng ạ